engine/src/main/java/org/terasology/world/block/Block.java

```diff
@@ -556,7 +598,8 @@ public void setLoweredLiquidMesh(Side side, BlockMeshPart meshPart) {
     }
 
     /**
-     * @param side
+     * Check is the side of this block is full side
+     * @param side checking side
      * @return Is the given side of the block "full" (a full square filling the side)
      */
     public boolean isFullSide(Side side) {
```

Comment:
no need for extra doc, the return statement documents that the given side is checked and there is not much more to add